docs/interface.md

```diff
@@ -0,0 +1,82 @@
+SauceLabs Interface
```

Comment:
The layout of this file is a little bit messed up, is this auto generated of from the console?

Reply:
yes and no .. I autogenerated most of it and modified it a bit so it looks better. After release we should have a script that autogenerates the API docs with better formatting. I think we can keep it for now .. it should just give an overview on what is available.

Reply:
Weel, to be honest, If people need to read if because that's the only docs, then it's not clear. I can try to fix it in this PR by pushing some changes

Reply:
I added a script that generates a nice table